src/Components/Publishing/Icon/IconPlus.tsx

```diff
@@ -33,8 +31,10 @@ export const IconPlus: React.SFC<Props> = props => {
 const StyledSvg = styled.svg`
   width: 15px;
   height: 15px;
+  margin: 0 20px;
```

Comment:
Margins seem contextual -- wouldn't it make more sense to do this outside the svg or add a prop?

Reply:
Good catch!